### ISSUE TITLE
axis format method broken (not called) since v0.49.1

### ISSUE THREAD

**Issue by stahlmanDesign** [plouc/nivo#331]
I noticed some values in my axes were showing milliseconds. I thought I had applied some formatting.

Sure enough, I have:
`axisLeft: {
  format: v => this.axisLeftFormat // <-- formats milliseconds to hh:mm:ss or adds a + '%' etc.
}`

In `"@nivo/bar": "^0.33.0"` the method is called and my 7000000 milliseconds show up as `1:56:40`

But in `"@nivo/bar": "^0.50.0"` the method is not called and it just displays 7000000

It seems to affect all the axes: axisLeft, axisTop, axisBottom etc

There's a storybook that shows this is the correct way to format an axis: http://nivo.rocks/storybook/?selectedKind=Bar&selectedStory=with%20formatted%20values&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs

The storybook code 'with formatted values' is here: https://github.com/plouc/nivo/blob/master/packages/bar/stories/bar.stories.js

This seems like a regression, or maybe I am missing the new way to do this and the documentation is not up to date?

**Comment by stahlmanDesign** [comment]
It works in v0.44.0

I'll try to hunt down the exact release that broke axis format

**Comment by plouc** [comment]
@stahlmanDesign, I have several package updates to be published along with the corresponding documentation, I hope I'll be able to publish this by the end of the week (end) if I find the time to do so.

I've updated several components' axes to use `@nivo/axes` instead of `@nivo/core` and the API changed a little.

````
## v0.49.1

#### Features

* **bar**  use @nivo/axes instead of @nivo/core for SVG axes ([3b22c6fb](https://github.com/plouc/nivo/commit/3b22c6fbfe553ee74e82481d7d7d443fe924a339))
* **examples**  upgrade retro example deps ([203f7198](https://github.com/plouc/nivo/commit/203f719840a0f0287542567e2ca4bec8de08a5a5))
* **website**  upgrade nivo packages ([531e492b](https://github.com/plouc/nivo/commit/531e492bea9b9d46a9633a2381dabbfe49584530))
```

I had renamed `format` to `tickFormat`, but reverted since as it's better to stay BC.

Sorry for the inconvenience, in the end axes should be more powerful, supporting more (d3) scale types.

**Comment by plouc** [comment]
@stahlmanDesign should be fixed if you upgrade packages to `0.51.0`

**Comment by stahlmanDesign** [comment]
Thanks, v0.51.0 restores the functionality in v0.48.0

I will look into migrating to the `@nivo/axes` package. Sorry I missed it or maybe I could have resolved the issue. But at least now there is backwards compatibility.
